MiniSim/Extensions/UserDefaults+Configuration.swift

```diff
@@ -27,6 +28,11 @@ extension UserDefaults {
         set { set(newValue, forKey: Keys.isOnboardingFinished) }
     }
     
+    @objc dynamic public var menuImage: String {
```

Comment:
Can you make it return `MenuImage` enum that you've added?

And also as the default let's use `MenuImage.iphone`

Reply:
Objective-C compliance will have to be removed from menuImage, but I was able to rework the observer to that.